Resources/assets/css/legacy.css

```diff
@@ -0,0 +1,68 @@
+/*
+ * Not used anymore ?
+ */
+div.sonata-ba-modal-edit-one-to-one td.sonata-ba-list-field-batch,
+div.sonata-ba-modal-edit-one-to-one div.sonata-ba-list-actions,
+div.sonata-ba-modal-edit-one-to-one th.sonata-ba-list-field-header-batch {
+    display: none;
+}
+
+div.sonata-ba-modal-edit-one-to-one div.sonata-ba-list-actions {
+    display: none;
+}
+
+
+legend.sonata-ba-fieldset-collapsed-description + .sonata-ba-collapsed-fields {
+    margin-top: 0;
+}
+
+.sonata-ba-fieldset-collapsed legend:before {
+    content: '+ ';
+}
+
+/*
+ * The .filter_legend class isn't used anywhere
```

Comment:
:) Here is what I did : `git log -S filter_legend -p Resources/views/CRUD/base_list.html.twig`